llvm/lib/Transforms/IPO/FunctionAttrs.cpp

```diff
@@ -71,7 +71,9 @@ using namespace llvm;
 #define DEBUG_TYPE "function-attrs"
 
 STATISTIC(NumMemoryAttr, "Number of functions with improved memory attribute");
-STATISTIC(NumNoCapture, "Number of arguments marked nocapture");
+STATISTIC(NumCapturesNone, "Number of arguments marked captures(none)");
+STATISTIC(NumCapturesOther, "Number of arguments marked with captures "
```

Comment:
"Other" is a bit overloaded here. First read I thought you meant other component. What about "Partial"?

Reply:
Done.